internal/runbits/requirements/requirements.go

```diff
@@ -406,3 +411,19 @@ func initializeProject() (*project.Project, error) {
 
 	return project.FromPath(target)
 }
+
+func versionStringToRequirements(version string) ([]bpModel.VersionRequirement, error) {
+	constraints, err := model.VersionStringToConstraints(version)
+	if err != nil {
+		return nil, errs.Wrap(err, "Unable to process version string into constraints")
+	}
+
+	requirements := make([]bpModel.VersionRequirement, len(constraints))
+	for i, constraint := range constraints {
+		requirements[i] = bpModel.VersionRequirement{
+			bpModel.VersionRequirementComparatorKey: constraint.Comparator,
+			bpModel.VersionRequirementVersionKey:    constraint.Version,
+		}
+	}
+	return requirements, nil
```

Comment:
I don't like that we're modeling this data outside of the model package. It feels like this should be owned by the model. We should use and enrich the params that model.StageCommit uses.